var/spack/repos/builtin/packages/quinoa/package.py

```diff
@@ -0,0 +1,63 @@
+##############################################################################
+# Copyright (c) 2017, Los Alamos National Security, LLC
+# Produced at the Los Alamos National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+
+from spack import *
+
+
+class Quinoa(CMakePackage):
+    """Quinoa is a set of computational tools that enables research and
+       numerical analysis in fluid dynamics. At this time it is a test-bed
+       to experiment with various algorithms using fully asynchronous runtime
+       systems.
+    """
+
+    homepage = "http://quinoacomputing.org"
+    url      = "https://github.com/quinoacomputing/quinoa/tarball/v1.0"
```

Comment:
@junghans, v1.0 does not exist, only v0.1, but that is also very old. The tags/releases so far really only mean some kind of development milestone, not really a real release. Can we just link the master branch?